src/pages/Home/Home.jsx

```diff
@@ -1,7 +1,22 @@
 import "./home.css";
 import { Link } from "react-router-dom";
+import axios from "axios";
+import { useState, useEffect } from "react";
 
 const Home = () => {
+    const [ productCategories, setProductCategories] = useState([]);
+    const getProductCategories = async() => {
+        try {
+            const productsData = await axios.get("/api/categories");
+            setProductCategories(productsData.data.categories);
+        } catch (error) {
+            console.error(error)
+        }
+    }
+
+    useEffect(() => {
+        getProductCategories()
```

Comment:
can write IIFE here instead of creating getProductCategories function separately(Suggestion)

Reply:
Sure.